ygot/diff.go

```diff
@@ -365,6 +365,11 @@ type DiffPathOpt struct {
 	// generated structs, which can result in duplication of list key leaves in
 	// the diff output.
 	MapToSinglePath bool
+	// IgnoreNewPaths specifies that only paths that are set in the original value are
+	// considered in the diff -- paths that exist only in the modified struct are disregarded
+	// by the diff. This means that the contents of the Update field of the returned
+	// Notification will only contain modifications to the original value.
+	IgnoreNewPaths bool
```

Comment:
I'd suggest `ModifiedOnly` or `OnlyModifiedFields` might be a clearer name.

Reply:
I'd worry that this would imply it only returns updated fields, not deleted ones. Would IgnoreAdditions be clearer?

Reply:
SGTM.

Reply:
Done.